modsecurity/cppbuild.sh

```diff
@@ -23,15 +23,16 @@ fi
 
 case $PLATFORM in
     linux-x86_64)
+        yum install -y pcre2-devel
```

Comment:
To be removed - in forked repo changes to .github/actions/deploy-centos/action.yml does not take any affect somewhy and original code was executed in the job